graphql/queryResolvers/getLessonMentors.ts

```diff
@@ -0,0 +1,27 @@
+import db from '../../helpers/dbload'
+
+const { User, UserLesson } = db
+
+type ArgsGetLessonMentors = {
+  lessonId: string
+}
+
+export const getLessonMentors = async (
+  _parent: void,
+  args: ArgsGetLessonMentors
+) => {
+  const { lessonId } = args
+  try {
+    let results = await UserLesson.findAll({
+      where: { lessonId },
+      include: [{ model: User }]
+    })
+
+    return results.map((result: any) => {
+      return result.User
+    })
+  } catch (err) {
+    console.error('Unable to fetch data: ', err)
```

Comment:
instead of console.log, you should `throw New Error(some message here)`
And then you can test to see if that error message displays in your tests